server/build_event_protocol/accumulator/accumulator.go

```diff
@@ -66,7 +66,7 @@ type BEValues struct {
 	sawWorkspaceStatusEvent bool
 	sawBuildMetadataEvent   bool
 	buildStartTime          time.Time
-	buildFinished           bool
+	exitCode                *string
```

Comment:
nit: consider making it a plain old string since it's returned in `BuildExitCode()` as a string anyway?

Reply:
I'm using it to indicate `BuildFinished` as well, though?

Reply:
missed that! lgtm